src/renderer/utils/InstallerConfiguration.ts

```diff
@@ -354,7 +354,7 @@ export interface Configuration {
 
 export class InstallerConfiguration {
   static async obtain(): Promise<Configuration> {
-    return this.fetchConfigurationFromCdn()
+    return this.loadConfigurationFromLocalStorage()
```

Comment:
please wait for #477 to implement a developer option to force the use of a local configuration or use https://github.com/flybywiresim/installer-data/pull/40 for testing.

Reply:
Whoops that change wasn’t meant to be committed, will revert.